src/main/java/it/pagopa/interop/probing/eservice/operations/mapstruct/mapper/MapStructMapper.java

```diff
@@ -9,7 +9,9 @@
 import it.pagopa.interop.probing.eservice.operations.dtos.ChangeEserviceStateRequest;
 import it.pagopa.interop.probing.eservice.operations.dtos.ChangeProbingFrequencyRequest;
 import it.pagopa.interop.probing.eservice.operations.dtos.ChangeProbingStateRequest;
+import it.pagopa.interop.probing.eservice.operations.dtos.EserviceSaveRequest;
 import it.pagopa.interop.probing.eservice.operations.dtos.EserviceViewDTO;
+import it.pagopa.interop.probing.eservice.operations.mapstruct.dto.SaveEserviceDto;
 import it.pagopa.interop.probing.eservice.operations.mapstruct.dto.UpdateEserviceFrequencyDto;
 import it.pagopa.interop.probing.eservice.operations.mapstruct.dto.UpdateEserviceProbingStateDto;
 import it.pagopa.interop.probing.eservice.operations.mapstruct.dto.UpdateEserviceStateDto;
```

Comment:
Mapstruct is a library, DTO is a model, please be agnostic. package is model not mapstruct and avoid to include the name of the library in file name as instance MapStructMapper is MapperImpl (maybe in the future you may change converter library)

Reply:
Okay, I fixed it